stable/vsphere-cpi/values.yaml

```diff
@@ -42,9 +42,10 @@ serviceAccount:
 daemonset:
   annotations: {}
   image: gcr.io/cloud-provider-vsphere/cpi/release/manager
-  tag: v1.1.0
+  tag: v1.2.0
   pullPolicy: IfNotPresent
   dnsPolicy: ClusterFirst
+  useHostNetwork: true
```

Comment:
Wondering if we should just remove this option? I don't think we would ever want host network to be false?

Reply:
Yea, I'm not sure what the use case would be... unless like some other CNI or Mesh Network needed it?? I can remove the option